Makefile

```diff
@@ -171,7 +171,7 @@ $(Worker_Enclave_u_Object): service/Enclave_u.o
 $(Worker_Name): $(Worker_Enclave_u_Object) $(SRC_Files)
 	@echo
 	@echo "Building the integritee-service"
-	@SGX_SDK=$(SGX_SDK) SGX_MODE=$(SGX_MODE) cargo build -p integritee-service $(Worker_Rust_Flags)
+	@SGX_SDK=$(SGX_SDK) SGX_MODE=$(SGX_MODE) cargo build -vv -p integritee-service $(Worker_Rust_Flags)
```

Comment:
Do we want to keep this? I guess this is a bit too verbose in the usual case.

Reply:
Nope, that was a leftover from when I was trying to debug the linker error, thanks!